content/applications/finance/accounting/fiscal_localizations/localizations/kenya.rst

```diff
@@ -0,0 +1,119 @@
+users and the internet is required.
+
+Installing the proxy server on a Windows device
+-----------------------------------------------
+
+Go to odoo.com/download, fill out the required information and click :guilabel:`Download`.
+
+.. image:: kenya/download.png
+   :align: center
+   :alt: Install the Proxy Server on a Windows device
+
+Once it is loaded on your computer, a wizard opens. You have to read and agree with the terms of the
+agreement. On the next page, select the :guilabel:`type of install: Odoo IoT. Then, click
+:guilabel:`Next` and :guilabel:`Install`. Once completed, click :guilabel:`Next`. Check the
+:guilabel:`Start Odoo` box to be redirected to Odoo automatically, and then click :guilabel:`Finish`.
+
+A new page opens, confirming your :doc:`IoT Box <../../../../productivity/iot/config/connect>` is up
+and running. Connect your physical device **Tremol G03 Control Unit (type C)** to your laptop via
+USB. In the :guilabel:`IoT Device` section, check that your Tremol G03 Control Unit (type C) appears.
+
+.. image:: kenya/iot-box.png
+   :align: center
+   :alt: Your IoT box is up and running
+
+.. note::
+   If the device is not detected, try to plug it in again or click on the :guilabel:`Restart` button
+   in the top right corner.
+
+.. seealso::
+   :doc:`Connect an IoT box to your database <../../../../productivity/iot/config/connect>`
+
+Sending the data to KRA using the Tremol G03 Control Unit
+---------------------------------------------------------
+
+As a pre-requisite, check out that the :ref:`Kenyan Accounting modules
+<localization/kenya/configuration>` are installed on your database. Then, go to
+:menuselection:`Accounting --> Configuration --> Settings --> Kenya TIMS Integration section`, and
+check that the :guilabel:`control Unit Proxy Address` has been automatically filled out, confirming
+the connection is established between the device and your computer.
+
+To send data to KRA, create a new invoice by going to :menuselection:`Accounting Dashboard -->
+Customer Invoice card` and click :guilabel:`New Invoice`. Upon confirmation of a new invoice, the
+:guilabel:`Send invoice to device` button appears. Clicking on it sends the invoice details to the
+device and from the device to the government. The :guilabel:`CU Invoice Number` field is now
+completed in your invoice, confirming the information has been sent.
+
+The :guilabel:`Tremol G03 Control Unit` tab contains fields that are automatically completed once
+the invoice is sent to the government:
+
+- :guilabel:`CU QR Code`: Url from the KRA portal which reflects a QR code.
+- :guilabel:`CU Serial Number`: reflects the serial number of the device.
+- :guilabel:`CU Signing Date and Time`: The date and time when the invoice has been sent to KRA.
+
+If you click on :guilabel:`Send and Print`, a .pdf of the invoice is generated. The
+:guilabel:`Kenyan Control Unit Info` is mentioned on the document.
```

Comment:
again, similar thing to above, but `Kenyan Control Unit Info` been changed to `Kenyan Fiscal Device Info` in that same PR